### DISCUSSION TITLE
Installing on ec2 Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by jayabrams** [openai/whisper#1406] [Q&A]
i can't for the life of my get whisper going on an ubuntu 22.04 machine from ec2

I have tried installing it with/without a virtual environment `venv` and still end up getting:
```ModuleNotFoundError: No module named 'whisper'```

Tried different python versions and followed some steps here `https://github.com/openai/whisper/discussions/403#discussioncomment-3949754` but no luck

When nothing else worked i tried `pip install --upgrade --no-deps --force-reinstall git+https://github.com/openai/whisper.git`
and then tried installing torch when it said `no module named torch` but still didn't work

Has anyone successfully been able to do this?
Thank you

**Comment by cyberfarer** [comment]
```
sudo apt install python3-full

cd ~
python3 -m venv whisper
cd whisper/bin/
./pip install -U openai-whisper

Successfully built openai-whisper lit
Installing collected packages: mpmath, lit, cmake, wheel, urllib3, typing-extensions, tqdm, sympy, regex, nvidia-nccl-cu11, nvidia-cufft-cu11, nvidia-cuda-nvrtc-cu11, numpy, networkx, more-itertools, MarkupSafe, llvmlite, idna, filelock, charset-normalizer, certifi, requests, nvidia-nvtx-cu11, nvidia-cusparse-cu11, nvidia-curand-cu11, nvidia-cuda-runtime-cu11, nvidia-cuda-cupti-cu11, nvidia-cublas-cu11, numba, jinja2, tiktoken, nvidia-cusolver-cu11, nvidia-cudnn-cu11, triton, torch, openai-whisper
Successfully installed MarkupSafe-2.1.3 certifi-2023.7.22 charset-normalizer-3.3.0 cmake-3.27.6 filelock-3.12.4 idna-3.4 jinja2-3.1.2 lit-17.0.2 llvmlite-0.41.0 more-itertools-10.1.0 mpmath-1.3.0 networkx-3.1 numba-0.58.0 numpy-1.25.2 nvidia-cublas-cu11-11.10.3.66 nvidia-cuda-cupti-cu11-11.7.101 nvidia-cuda-nvrtc-cu11-11.7.99 nvidia-cuda-runtime-cu11-11.7.99 nvidia-cudnn-cu11-8.5.0.96 nvidia-cufft-cu11-10.9.0.58 nvidia-curand-cu11-10.2.10.91 nvidia-cusolver-cu11-11.4.0.1 nvidia-cusparse-cu11-11.7.4.91 nvidia-nccl-cu11-2.14.3 nvidia-nvtx-cu11-11.7.91 openai-whisper-20230918 regex-2023.10.3 requests-2.31.0 sympy-1.12 tiktoken-0.3.3 torch-2.0.1 tqdm-4.66.1 triton-2.0.0 typing-extensions-4.8.0 urllib3-2.0.6 wheel-0.41.2
```

**Reply by cyberfarer** [reply]
I don't know if I need to change into the directory to run the command, because I was in too much of  a hurry to do it right the first time. :)

**Comment by MiManjaka42** [comment]
I have the same probleme on my ubuntu 22.04 too . Pls , have you solve it ?
